qiskit/aqua/circuits/fourier_transform_circuits.py

```diff
@@ -86,6 +92,32 @@ def construct_circuit(
             FourierTransformCircuits._do_swaps(circuit, qubits)
 
         qubit_range = reversed(range(len(qubits))) if inverse else range(len(qubits))
+
+        if precedes_measurement and not do_swaps and approximation_degree == 0:
+            if not inverse:
+                for j in qubit_range:
+                    circuit.add_register(ClassicalRegister(1))
+                    neighbor_range = range(np.max([0, j - len(qubits) + 1]), len(qubits) - j - 1)
+                    circuit.h(j)
+                    circuit.measure(j, j)
+                    for k in neighbor_range:
+                        lam = 1.0 * np.pi / float(2 ** (k + 1))
```

Comment:
You don't need to multiply with `1.0`, nor a conversion to float here, or the extra-brackets around `k`. Python 3 works does float arithmetics anyways (and on top you already use the float number `np.pi`).